server/start.go

```diff
@@ -36,7 +42,7 @@ const (
 
 // StartCmd runs the service passed in, either stand-alone or in-process with
 // Tendermint.
-func StartCmd(ctx *Context, appCreator AppCreator) *cobra.Command {
+func StartCmd(ctx *Context, cdc *codec.Codec, appCreator AppCreator, register api.RegisterRoutesFn) *cobra.Command {
```

Comment:
q: can we use a marshaler codec instead, or is this something we need to add to #6194?

Reply:
Yeah, we should use `WithJSONMarshaler`. I'll update.